ui-lib/global.css

```diff
@@ -0,0 +1,53 @@
+body {
```

Comment:
I know this file is ported from current `renderer/global.css` but do you know why it's not included in `createGlobalStyle`?